SU2_CFD/src/solvers/CSpeciesSolver.cpp

```diff
@@ -390,7 +396,6 @@ void CSpeciesSolver::BC_Inlet(CGeometry* geometry, CSolver** solver_container, C
       LinSysRes.AddBlock(iPoint, residual);
 
       /*--- Jacobian contribution for implicit integration ---*/
-      const bool implicit = (config->GetKind_TimeIntScheme() == EULER_IMPLICIT);
```

Comment:
revert all of these please, we want to be able to switch between explicit and implicit dynamically.

Reply:
54a0c69 Again I apologize to not use your suggestion but making it locally, gives me at least the chance to check compilation before pushing. I just checked how can add you as a [commit contributer](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) locally

Reply:
just gave it  a test 04533a5 there. So now I can add you as a co author whenever I dont feel like using the github suggestion. Credit where Credit's due 💵

Reply:
lol, thanks you don't have to do that, well I guess it's useful if it turns out to be a bug xD